js/ui/context_menu/ui.menu_base.js

```diff
@@ -534,17 +534,17 @@ class MenuBase extends HierarchicalCollectionWidget {
         const selectedIndex = this._dataAdapter.getSelectedNodesKeys();
 
         if(!selectedIndex.length || !this._selectedGetter(args.itemData) || !this._isItemSelectable(args.itemData)) {
-            this._setAriaSelected($itemElement, 'false');
+            this._setAriaSelectionAttribute($itemElement, 'false');
```

Comment:
How it works if `_setAriaSelectionAttribute` is empty function? Maybe this code is not need?

Reply:
this is just a renaming )

Reply:
It can be done in the separate pull request then. I think if we see some strange code near we should fix it or to understand how it work. Especially when it not need many time to research. And this case seems can be checked in 10 minutes and refactor in 1 minute. And the code will looks more better.